presto-main/src/main/java/io/prestosql/operator/aggregation/builder/SpillableHashAggregationBuilder.java

```diff
@@ -219,6 +210,15 @@ private boolean shouldMergeWithMemory(long memorySize)
         }
     }
 
+    private long getSizeInMemoryWhenUnspilling()
```

Comment:
I would keep the old method name `getSizeInMemory`. 
- the fact that it's only invoked after spiller is created seems not relevant to semantics of the method
- the method is invoked before spilling (ie before `getFutureValue(spillToDisk());`), so it's not "when unspilling"

Reply:
> the method is invoked before spilling (ie before getFutureValue(spillToDisk());), so it's not "when unspilling"

It is used to estimate size of memory during unspilling phase and to decide if we should merge with mem or not. This method has very limited use case and the previous name was confusing